nym-vpn-x/src-tauri/src/commands/node_location.rs

```diff
@@ -88,22 +77,16 @@ pub async fn get_node_location(
 
 #[instrument]
 #[tauri::command]
-pub async fn get_countries(node_type: NodeType) -> Result<Vec<Country>, CmdError> {
+pub async fn get_countries(node_type: NodeType) -> Result<Vec<Country>, BkdError> {
     debug!("get_countries");
     match node_type {
         NodeType::Entry => get_gateway_countries(false).await.map_err(|e| {
             error!("failed to get node locations: {}", e);
-            CmdError::new(
-                CmdErrorSource::InternalError,
-                "failed to get node locations",
-            )
+            BkdError::new_internal("failed to get node locations", None)
         }),
         NodeType::Exit => get_gateway_countries(true).await.map_err(|e| {
             error!("failed to get node locations: {}", e);
-            CmdError::new(
-                CmdErrorSource::InternalError,
-                "failed to get node locations",
-            )
+            BkdError::new_internal("failed to get node locations", None)
```

Comment:
I see multiple places where the underlying error is discarded. I think you'll need to include them, otherwise it's going to be hard for the user and us to understand what's going wrong and fix things

Reply:
This is voluntary. If db write throws error for some reason, we don't want to report the error tech details to the user. He's just not the right target for it. But the error will be "friendly" reported as "Oups an internal error occured" in the UI. And the tech details of the error will be:
- logged in output/logs (so as dev we should have mats to debug)
- reported by sentry (if turned on, feature planned)

Reply:
See my other comment why I disagree with this. Realistically speaking not including them will mean that for almost all bug reports from users we wont be able to do anything to fix them